MPCAutofill/cardpicker/templates/cardpicker/base.html

```diff
@@ -18,56 +18,74 @@
+        integrity="sha512-0bEtK0USNd96MnO4XhH8jhv3nyRF0eK87pJke6pkYf3cM0uDIhNJy9ltuzqgypoIFXw3JSuiy04tVk4AjpZdZw=="
+        crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"
+        integrity="sha512-hCP3piYGSBPqnXypdKxKPSOzBHF75oU8wQ81a6OiGXHFMeKs9/8ChbgYl7pUvwImXJb03N4bs1o1DzmbokeeFw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-fQybjgWLrvvRgtW6bFlB7jaZrFsaBXjsOMm/tB9LTS58ONXgqbR9W8oWht/amnpF"
+        crossorigin="anonymous"></script>
     <script src="{% static 'js/base.js' %}"></script>
-
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
 </head>
 <body id="id_body" style="touch-action: manipulation;">
 {# Navbar #}
-<nav class="navbar navbar-expand-md navbar-light bg-primary fixed-top">
-    <div class="container-fluid">
+<div class="navbar navbar-expand-lg fixed-top navbar-dark bg-primary" style="height: 50px;">
+    <div class="container justify-content-center fixed-top align-middle bg-primary">
         <a class="navbar-brand" href="/">
-            <img src="{% static 'cardpicker/logolowres.png' %}?10" width="30" height="30">
-            <b>MPC Autofill</b></a>
-        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+            <img src="{% static 'cardpicker/logolowres.png' %}?10" width="40" height="40" style="margin-right: 5px;">
+            <span class="align-middle"><b>{{ SITE_NAME }}</b></span>
+        </a>
+        <button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
                 aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
             <span class="navbar-toggler-icon"></span>
         </button>
-        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <div class="navbar-collapse collapse" id="navbarSupportedContent" style="font-weight: bold;">
             <ul class="navbar-nav mr-auto">
-                <li class="nav-item"><a class="nav-link" href="/guide">Guide</a></li>
-                <li class="nav-item"><a class="nav-link" href="/new">What's New?</a></li>
-                <li class="nav-item"><a class="nav-link" href="/credits">Contributions</a></li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/new">What's New?</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/credits">Credits</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/guide">Guide</a>
+                </li>

```

Comment:
personally prefer to keep the original order of guide -> what's new -> contributions here

Reply:
Done!